modules/jooby-metrics/src/main/java/io/jooby/internal/metrics/NoCacheHeader.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Jooby https://jooby.io
+ * Apache License Version 2.0 https://jooby.io/LICENSE.txt
+ * Copyright 2014 Edgar Espina
+ */
+package io.jooby.internal.metrics;
+
+import io.jooby.Context;
+
+public class NoCacheHeader {
+
+  public static void add(Context ctx) {
+    ctx.setResponseHeader("Cache-Control", "must-revalidate,no-cache,no-store");
```

Comment:
I don't like a class for adding a header. Can we add the header value as constant somewhere? Also, not sure we need it.